.jshintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "bitwise": false,
+  "freeze": true,
+  "globals": false,
+  "maxdepth": 3,
+  "maxparams": 4,
+  "maxstatements": 25,
+  "unused": true,
+  "varstmt": true,
```

Comment:
Causes lots of "`var` declarations are forbidden. Use `let` or `const` instead" errors